lib/gears/timer.lua

```diff
@@ -105,6 +105,8 @@ local timer_instance_mt = {
 -- @tparam table args Arguments.
 -- @tparam number args.timeout Timeout in seconds (e.g. 1.5).
 -- @tparam[opt=false] boolean args.autostart Automatically start the timer.
+-- @tparam function args.callback Shortcut to avoid having to
+--  use `connect_signal`.
```

Comment:
That is not really a good documentation. How about `Callback function to register. Equivalent to connecting to the "timeout" signal of the returned object.` or something like that.